src/core/product-transformer.ts

```diff
@@ -71,6 +71,35 @@ namespace Utils {
       return Utils.remap(transformed, effectiveStructure, baseProduct);
     };
   }
+
+  export const transformData = (data: any) => {
+    return ({
+      ...data,
+      price: transformPrice(data.price),
+    });
+  };
+
+  // note that this assumes data won't have thousands separators
+  export const transformPrice = (price: string): string => {
+    if (!price) {
+      return '';
+    }
+    const PRICE_REGEX = /^(.*)([\.,])(.*)$/;
+    const match = PRICE_REGEX.exec(price);
+    if (match) {
+      return match[1] + match[2] + simplePad(match[3].substr(0,2));
```

Comment:
Also just double checking is there any case match[3] is undefined?

Reply:
I would actually be in favour of using a ternary expression here (`match ? ... : ...`). `match[3]` won't be `undefined` if there is a match because it can always match the empty string.

Reply:
No, if match is not null then the regex is bound to match. Something like "12." will just have the third group as the empty string.